src/coreclr/jit/lclvars.cpp

```diff
@@ -334,19 +334,6 @@ void Compiler::lvaInitTypeRef()
         {
             LclVarDsc* const varDsc = lvaGetDesc(lclNum);
             varDsc->lvIsOSRLocal    = true;
-
-            if (info.compPatchpointInfo->IsExposed(lclNum))
-            {
```

Comment:
Do we still need to communicate the exposure information in the patchpoint information?

Reply:
No. However, I'll leave this as is for now so older jits can still do somewhat reasonable things with newer SPMI data.